include/dkm_utils.hpp

```diff
@@ -44,85 +75,53 @@ std::vector<std::array<T, N>> get_cluster(const std::vector<std::array<T, N>>& p
  *
  * @param points Sequence that were passed to dkm::kmeans_lloyd
  * @param points Result of dkm::kmeans_lloyd
```

Comment:
I think the parameter name in this comment is incorrect? Should this be `means`?